ndc-client/src/models.rs

```diff
@@ -60,6 +61,17 @@ pub struct QueryCapabilities {
 }
 // ANCHOR_END: QueryCapabilities
 
+// ANCHOR: MutationCapabilities
+#[skip_serializing_none]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[schemars(title = "Mutation Capabilities")]
+pub struct MutationCapabilities {
+    /// Whether or not nested inserts to related collections are supported
+    pub nested_inserts: Option<LeafCapability>,
```

Comment:
Should we consider leaving all nested stuff out of the first iteration of mutations entirely, with an eye on looking to a nicer way to compose smaller operations together?